site/_data/deprecation-calendar/deprecation-articles.yml

```diff
@@ -0,0 +1,2 @@
+- deprecation_id: 6239658726391808
```

Comment:
Since you got `deprecations` (mind the s 😉 ) in the folder title, no need to duplicate it in the file name.